src/tools/compiletest/src/lib.rs

```diff
@@ -832,41 +834,28 @@ fn find_tests_in_dir(
         let file_path = Utf8PathBuf::try_from(file.path()).unwrap();
         let file_name = file_path.file_name().unwrap();
 
-<<<<<<< HEAD
         if is_test(&file_name) && (!config.only_modified || modified_tests.contains(&file_path)) {
-=======
-        if is_test(file_name)
-            && (!cx.config.only_modified || cx.modified_tests.contains(&file_path))
-        {
->>>>>>> pull-upstream-temp--do-not-use-for-real-code
             // We found a test file, so create the corresponding libtest structures.
             debug!(%file_path, "found test file");
 
             // Record the stem of the test file, to check for overlaps later.
-            let rel_test_path = relative_dir_path.join(file_path.file_stem().unwrap());
+            let rel_test_path = Utf8PathBuf::try_from(relative_dir_path.join(file_path.file_stem().unwrap()));
             let paths =
-                TestPaths { file: file_path, relative_dir: relative_dir_path.to_path_buf() };
+                TestPaths { file: file_path, relative_dir: (&relative_dir_path).into() };
 
-            on_test_found(&paths, Some(rel_test_path));
+            on_test_found(&paths, Some(rel_test_path.unwrap()));
         } else if file_path.is_dir() {
             // Recurse to find more tests in a subdirectory.
-<<<<<<< HEAD
-            let relative_file_path = relative_dir_path.join(file.file_name());
-            if &file_name != "auxiliary" {
-                debug!("found directory: {:?}", file_path.display());
+            let relative_file_path = relative_dir_path.join(file.file_name().to_str().unwrap());
+            if file_name != "auxiliary" {
+                debug!("found directory: {:?}", file_path.as_std_path().display());
                 find_tests_in_dir(
                     config.clone(),
                     &file_path,
                     &relative_file_path,
                     modified_tests,
                     on_test_found,
                 )?;
-=======
-            let relative_file_path = relative_dir_path.join(file_name);
-            if file_name != "auxiliary" {
-                debug!(%file_path, "found directory");
```

Comment:
we lost this in your change, which looks like a tracing thing

Reply:
It is above
```rust
            let relative_file_path = relative_dir_path.join(file.file_name().to_str().unwrap());
            if file_name != "auxiliary" {
                debug!("found directory: {:?}", file_path.as_std_path().display());